modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,451 @@
+
+* With this update, several potential causes for {tekton-results} becoming unresponsive, crashing, or consuming a large amount of memory were removed.
+
+* Before this update, in the Pipeline details page of the web console, if a `when` expression using CEL was configured for a task, information was not displayed correctly. With this update, the information is displayed correctly.
+
+* Before this update, in the Pipeline details page of the web console, the menu was not visible when you enabled dark mode in the web console. With this update, the menu is visible.
+
+* Before this update, in the Pipelines page of the web console, information about running statistics of pipelines did not include the information saved in {tekton-results}. With this update, the page includes all available running statistics information for every pipeline.
+
+* Before this update, when you viewed a list of tasks for a namespace in the web console, a task from another namespace was sometimes displayed in the list. With this update, the web console correctly lists tasks for each namespace.
+
+* Before this update, when you viewed the list of task runs in the web console, the status for each task run was not displayed. With this update, the list of task runs in the web console includes the status for each task run.
+
+* Before this update, if you disabled cluster tasks in your {pipelines-shortname} deployment, the Pipeline Builder in the web console did not work. With this update, if you disable cluster tasks, the Pipeline Builder in the web console works correctly.
+
+* Before this update, the {pipelines-shortname} console plugin pod did not move to the node specified using the `nodeSelector`, `tolerations`, and `priorityClassName` settings. With this update, the {pipelines-shortname} plugin pod moves to the correct node.
+
+* Before this update, the {pac} watcher sometimes crashed when processing a pipeline run for which a concurrency limit was not set. With this update, these pipeline runs are processed correctly.
+
+* Before this update, in {pac}, a concurrency limit setting of `0` was not interpreted as disabling the concurrency limit. With this update, a concurrency limit setting of `0` disables the concurrency limit.
+
+* Before this update, when you defined annotations and labels for a task in {pac}, the annotations and labels were not set on the pod that is running the task. With this update, {pac} correctly sets the configured annotations and labels on the pod that is running the task.
+
+* Before this update, {pac} sometimes caused a load on the Kubernetes service by re-reading an internal configuration `ConfigMap` frequently. With this update, {pac} no longer causes this load, because it reloads the `ConfigMap` only after the `ConfigMap` is modified.
```

Comment:
```suggestion
* Before this update, {pac} sometimes caused a load on the Kubernetes service by re-reading an internal configuration `ConfigMap` resource frequently. With this update, {pac} no longer causes this load, because it reloads the `ConfigMap` only after the `ConfigMap` is modified.
```